about.php

```diff
@@ -86,6 +117,125 @@
+                    <div class="modal-content modal-dialog-centered">
+                        <div class="modal-header">
+                            <form class="row">
+                                <h5 class="modal-title">Category Crud</h5>
+                                <button type="button" id="addCategorybtn" class="btn btn-light">Add</button>
+                                <button type="button" id="updateCategorybtn" class="btn btn-light">Update</button>
+                                <button type="button" id="deleteCategorybtn" class="btn btn-light">Delete</button>
+                            </form>
+                        </div>
+                        <div class="modal-body modal-dialog-scrollable">
+                            
+                            <form id="addCategoryForm" method="post">
+                                <h5 class="modal-title">Add New Category</h5>
+                                <input type="text" class="btn-light" name="addCategorytxt">
+                                <input type="submit" class="btn btn-primary" value="Add" name="addCategory">
+                            </form>
+
+                            <form id="updateCategotyForm">
+                                <h5 class="modal-title">Update Category</h5>
+                                <ul class="list-group list-group-flush">
+                                    
+                                    <?php 
+                                
+                                    $categories = $data->select_categories();
+                                    for ($a=0; $a < count($categories); $a++){                                                                                  
+                                    ?>    
+                                    <li class="list-group-item" class="">
+                                        <button type="button" id="updatebtn"  update_name="<?php echo $categories[$a]["name"]; ?>" update_id="<?php echo $categories[$a]["id"];?>" class="btn btn-light fa fa-check float-right" data-toggle="modal" data-target="#updateConfirmModal" data-dismiss="modal"></button>
+                                        <?php echo $categories[$a]["name"]; ?>
+                                    </li>
+                                    <?php 
+                                    }
+                                    ?>
+                                </ul>
+                            </form>
+
+                            <form id="deleteCategotyForm">
+                                <h5 class="modal-title">Delete Category</h5>
+                                <ul class="list-group list-group-flush">
+                                    
+                                    <?php 
+                            
+                                    for ($a=0; $a < count($categories); $a++){                                                                                  
```

Comment:
pwede ka gumagamit dito ng `foreach` para maiwasan mo gumagamit ng combination ng `for` at `count`